treeherder/etl/perf.py

```diff
@@ -119,7 +119,7 @@ def load_perf_artifacts(project_name, reference_data, job_data, datum):
                     'platform': platform,
                     'framework': framework,
                     'extra_properties': extra_properties,
-                    'last_updated': push_timestamp
```

Comment:
Since we changed to update_or_create, the series will now update the existing series' properties that in the default dict every time that function is called. If we updated last_updated here it would mean that the values for that column might occasionally *go down* (if we're ingesting an older push) which isn't what we want. Fortunately, since we always update laste_updated if necessary in PerformanceDatum (https://github.com/mozilla/treeherder/blob/master/treeherder/perf/models.py#L81) this isn't necessary. :poodle: